ModTek/ModTek.cs

```diff
@@ -143,6 +143,21 @@ public static void Init()
             stopwatch.Stop();
         }
 
+        public static void Cleanup()
+        {
+            CloseLogStream();
+
+            modLoadOrder = null;
```

Comment:
what about moving all those variables you are setting to null to a composite structure and then only set that one structure to null? the real proper solution would be to get rid of static classes as much as possible, so moving all logic to a single instanced ModTekController class that gets set to null once finished

Reply:
I personally don't think I like classes that are instantiated only a single time for a single purpose -- it feels like a breakdown of OO. However, I agree that this sort of cleanup step sort of underlines a problem of using static members as "fake" parameters. I'll take a look at what it would take to move that way.

Reply:
The builder pattern is often used for that, something that is used to build something and then thrown away. In this case the overriding manifest.